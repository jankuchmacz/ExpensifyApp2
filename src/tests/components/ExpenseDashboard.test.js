import React from "react";
import ExpenseDashboardPage from "../../components/Dashboard";
import { shallow } from "enzyme";

test('should render ExpenseDasboard Page page correctly', ()=>{
    const wrapper = shallow(<ExpenseDashboardPage/>);
    expect(wrapper).toMatchSnapshot();
})