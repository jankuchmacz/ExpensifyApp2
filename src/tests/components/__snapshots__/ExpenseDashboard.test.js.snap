// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDasboard Page page correctly 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
